packages/web/src/executors/rollup/schema.json

```diff
@@ -119,10 +119,19 @@
       "type": "boolean",
       "description": "Sets `javascriptEnabled` option for less loader",
       "default": false
+    },
+    "generateExportsField": {
+      "type": "boolean",
+      "description": "Generate package.json with 'exports' field. This field defines entry points in the package and is used by Node and the TypeScript compiler.",
+      "default": false
+    },
+    "skipTypeField": {
```

Comment:
question: not seeing this `skipTypeField` being used in `update-package-json.ts`, is this intended?